src/pages/docs/platform/auth/capabilities.mdx

```diff
@@ -0,0 +1,897 @@
+---
+title: Identified clients and capabilities
+meta_description: "Comprehensive guide to Ably's permissions system including API key capabilities, client identity management, and fine-grained access control."
+redirect_from:
+  - /docs/auth/capabilities
+  - /docs/auth/identified-clients
+---
+
+## Identified clients
```

Comment:
This is very passive in tone. Id go with "Clients are anonymous by default etc."... then explain the rest..